### PR TITLE
added Makeover Game

### LINKED ISSUE #4603: [New game]:  Makeover Game
### 🎮 Game Request

In this user can style the character in various dresses.

### Point down the features

User can style:
1. Hair
2. Skin Type
3.  Dresses
4. Shoes
And when finished can take a picture and save it to their local computer.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by abckhush**
## PR Description 📜

added Makeover_Game
Fixes #4603

<hr>
 
## Mark the task you have completed ✅

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [X] I have added screenshot for website preview in assets/images 
- [X] I have added entries for my game in main README.md
- [X] I have added README.md in my folder 
- [X] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Makeover_Game

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @abckhush ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by abckhush** [comment]
Hi @ishita-43. Kindly review this PR.

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please addd your game image in asset folder of the gamezone and follow naming convention while naming that image

**Comment by abckhush** [comment]
Hi @ishita-43  I have added, kindly check.

File: assets/images/Screenshot 2024-06-19 001746.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-06-19 001746.png:1]: rename the image same as your game folder name follow contribution guidelines for naming convention

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: assets/images/Screenshot 2024-06-19 001746.png
Comment (abckhush) [reply assets/images/Screenshot 2024-06-19 001746.png:1]: I am sorry for the overlook, can you check now?

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @abckhush , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**